e2e/test/scripts/create_cluster.sh

```diff
@@ -22,7 +23,7 @@ variable "server_type_master" {
   default = "g6-standard-2"
 }
 variable "region" {
-  default = "eu-west"
+  default = "$REGION"
```

Comment:
Again to not break any existing automation maybe we can look to check if `REGION` is not set. If not set then keep this default. If it is set then override it with what was passed in.

Reply:
Done, thanks for the suggestion.